### PR TITLE
Update graph in README

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
 # Change Log
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
+
+## 0.3.0 - 2017-12-15
+### Fixed
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md]: I prefer the 'Added' section above 'Fixed' (it's the more exciting stuff)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+
+### Added
+- Graph:
+  - Customize the color and display text of vertices through VS Code [settings](README.md#graphSettings)
+  - Automatically choose vertex color based on label
+  - Honor current VS Code theme
+
 ## 0.2.2 - 2017-12-04
 ### Added
-- View, edit, and persist a Mongo collection
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md]: Why did you remove this line?

Comment (StephenWeatherford) [reply CHANGELOG.md]: Because I'm an idiot.

File: CHANGELOG.md (hunk 1 above)
Comment (StephenWeatherford) [reply CHANGELOG.md]: Fixed Fixed. :-)
